Translated/Quests/Cocoons/Setsuna no Issen/quest_01100080.csv

```diff
@@ -1,8 +1,8 @@
 a1#0,"""特定のクライアントオーダーのクリア"""
 a2#0,"""特定のクライアントオーダーのクリア"""
 a3#0,"""スーパーハード受注許可証の所持"""
-Area0S#0,"""Split-second Rush"""
-Area0L#0,"""Split-second Rush"""
-Title#0,"""Split-second Rush"""
+Area0S#0,"""Momentary Flash"""
+Area0L#0,"""Momentary Flash"""
+Title#0,"""Momentary Flash"""
```

Comment:
This misses the point of what Setsuna no Issen would translate to. If we were to take into account of what it would translate in its literal meaning, then this would be correct. However, it's referring to a movement in the blink of an eye, which is what most RPGs would refer to as the "Blink" mechanic.

Reply:
Why not translate it as "In the Blink of an Eye"?